src/java/htsjdk/variant/variantcontext/VariantContextUtils.java

```diff
@@ -338,6 +338,55 @@ public static boolean match(VariantContext vc, Genotype g, JexlVCMatchExp exp) {
     }
 
     /**
+     * Answers if the provided variant is transitional (otherwise, it's transversional).
+     * Transitions:
+     * A->G
+     * G->A
+     * C->T
+     * T->C
+     * <p/>
+     * Transversions:
+     * A->C
+     * A->T
+     * C->A
+     * C->G
+     * G->C
+     * G->T
+     * T->A
+     * T->G
+     *
+     * @param vc a biallelic polymorphic SNP
+     * @return true if a transition and false if transversion
+     * @throws IllegalArgumentException if vc is monomorphic, not a SNP or not bi-allelic.
+
+          */
+
+    static public boolean isTransition(final VariantContext vc) throws IllegalAccessException {
```

Comment:
FYI this is already implemented in GATKVariantContextUtils.java (in the GATK obviously).  We only moved half of the utility code over to htsjdk and kept the other half in the GATK.  You might want to look at the implementation in the GATK and move that over here instead.